sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import nodeResolve from "rollup-plugin-node-resolve";
+import multiEntry from "rollup-plugin-multi-entry";
+import cjs from "rollup-plugin-commonjs";
+import json from "rollup-plugin-json";
+import replace from "rollup-plugin-replace";
+import { uglify } from "rollup-plugin-uglify";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import shim from "rollup-plugin-shim";
+// import visualizer from "rollup-plugin-visualizer";
+
+const version = require("./package.json").version;
+const banner = [
+  "/*!",
+  ` * Azure Storage SDK for JavaScript - Blob, ${version}`,
+  " * Copyright (c) Microsoft and contributors. All rights reserved.",
+  " */"
+].join("\n");
+
+const pkg = require("./package.json");
+const depNames = Object.keys(pkg.dependencies);
+const production = process.env.NODE_ENV === "production";
+
+export function nodeConfig(test = false) {
+  const externalNodeBuiltins = [
+    "@azure/ms-rest-js",
+    "crypto",
+    "fs",
+    "events",
+    "os",
+    "stream"
+  ];
+  const baseConfig = {
+    input: "dist-esm/src/index.js",
+    external: depNames.concat(externalNodeBuiltins),
+    output: {
+      file: "dist/index.js",
+      format: "cjs",
+      sourcemap: true
+    },
+    plugins: [
+      sourcemaps(),
+      replace({
+        delimiters: ["", ""],
+        values: {
+          // replace dynamic checks with if (true) since this is for node only.
+          // Allows rollup's dead code elimination to be more aggressive.
+          "if (isNode)": "if (true)"
+        }
+      }),
+      nodeResolve({ preferBuiltins: true }),
+      cjs(),
+      json()
```

Comment:
Is "json" plugin necessary?

Reply:
From my understanding, `rollup-plugin-json` is not necessary.
Removed it - https://github.com/Azure/azure-sdk-for-js/pull/2297/commits/00f33e09bbb2066c003e90081dd3147574582745